test/index.js

```diff
@@ -35,10 +35,10 @@ test('composite selector', async t => {
 test('should throw exception for non-valid selectors', async t => {
     for (const selector of [null, false, {}, 42]) {
         try {
-            VueSelector(selector);
+            await VueSelector(selector);
```

Comment:
As I see from the description this selector should always raise an error. Actually the test will pass without any assertions if the `await VueSelector(selector)` doesn't throw an error. You should write it in the following way:
```js
try {
    await VueSelector(selector);
    throw new Error('The selector should throw but it doesn't.');
}
catch (e) {
    //...
}
```